### ISSUE TITLE
[cms-v2] Master issue for coordination and checklist

### ISSUE THREAD

**Issue by fernandoabolafio** [decred/politeiagui#1604]
## CMS refactor coordination issue

This issue intends to describe the work to be done for the competition of the "CMS v2". 
Different from the work done for the proposals website, the CMS won't have design specs prior to the development. Thus, it is up to the developers to follow the layout and code style employed on the proposals redesign.

## Work Summary and checklist

The following is a checklist containing the work to be done for the refactoring:

- [ ] Authentication 
     - [ ] Login
     - [ ] Signup
     - [ ] Reset password
     - [ ] Resend verification email
- [ ] Invoice
    - [ ] User List
    - [ ] Admin List
    - [ ] New
    - [ ] Edit
    - [ ] Draft
    - [ ] Detail
    - [ ] Admin actions (approve, reject, dispute)
    - [ ] Comments
- [ ] Payouts
    - [ ] Generate payouts
    - [ ] Payouts summary
- [ ] User/Contractor 
    - [ ] Manage tab
    - [ ] Invite
    - [ ] Search
- [ ] DCC
     - [ ] List
     - [ ] New DCC
     - [ ] Detail
     - [ ] Admin review (approve/reject)
     - [ ] Support/opose

## Methodology

In order to accomplish the work described above and keep the current website working, this work will be completed in a separated branch which will be constantly synchronized with the master branch. Once this refactoring is done, there should not be any backwards compatibility with the old side. Meaning that all remaining artifacts and libraries ("snew-classic-ui" for instance) will be removed.

**Comment by fernandoabolafio** [comment]
duplicated of https://github.com/decred/politeiagui/issues/1685

**Comment by xaur** [comment]
Actually #1685 is a duplicate of this since it was created later. Please don't recreate issues because it breaks existing links and makes navigation harder. e.g. #1625 still says "to accomplish what is described in #1604".
